civihr_default_theme/scss/components/_header.scss

```diff
@@ -215,16 +221,23 @@
     .menu-link-hr-admin {
       border: 1px solid $gray-darker;
       line-height: 50px;
-      margin: 25px;
+      margin: 25px auto;
       padding: 0;
       text-align: center;
+      width: 300px;
```

Comment:
I don't think we should set a fixed width, can we make it `display: inline-block` instead?

Reply:
Removing the width completely works too. Please check the updated screenshots.